bigquery/google/cloud/bigquery/job.py

```diff
@@ -1243,7 +1279,8 @@ def __init__(self, job_id, query, client,
     https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.allowLargeResults
     """
 
-    create_disposition = CreateDisposition('create_disposition')
+    create_disposition = CreateDisposition('create_disposition',
```

Comment:
Does this mean the QueryJob needs a `self._properties = {}` in the constructor now?

Reply:
I thought we were doing the `ApiResourceProperty` thing for all the job config classes? I was copying `ExtractJobConfig`. I don't have an opinion about this&mdash;we can do whatever makes the most sense to you.

Reply:
We are, but I saw this is on QueryJob, which hasn't been fully converted yet. If the system tests seem okay with it, I think it's fine to leave as you have it until we convert QueryJob + QueryJobConfig, too.